tests/e2e/tests-library/ProjectAndFileTests.ts

```diff
@@ -93,6 +94,39 @@ export class ProjectAndFileTests {
 		}
 	}
 
+	/**
+	 * perform to 'Trust Publisher' dialog box, when it appears
+	 */
+	async performTrustPublisherDialog(): Promise<void> {
+		Logger.debug();
+
+		try {
+			// add TS_IDE_LOAD_TIMEOUT timeout for waiting for appearing the box
+			await this.driverHelper.wait(TIMEOUT_CONSTANTS.TS_IDE_LOAD_TIMEOUT);
+			await this.driverHelper.waitVisibility(
+				ProjectAndFileTests.TRUST_PUBLISHER_BOX,
+				TIMEOUT_CONSTANTS.TS_SELENIUM_CLICK_ON_VISIBLE_ITEM
+			);
+			await this.driverHelper.waitAndClick(
+				this.cheCodeLocatorLoader.webCheCodeLocators.WelcomeContent.button,
+				TIMEOUT_CONSTANTS.TS_SELENIUM_CLICK_ON_VISIBLE_ITEM
+			);
+		} catch (e) {
+			Logger.info('"Do you trust the publisher?" dialog box was not shown');
+		}
+	}
+
+	/**
+	 * perform to 'Trust Authors' dialog box, when it appears
+	 * perform to 'Trust Publisher' dialog box, when it appears
+	 */
+	async performTrustAuthorAndPublisherDialog(): Promise<void> {
```

Comment:
I would propose more flexible method name `performTrustDialogs()`, which will consume other possible "Trust..." popups in future.

Reply:
Yeah..makes sense..
Will fix it, thanks

Reply:
thank you for the fix